src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/MessageInterception/InterceptMethodAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Utilities;
+
+#nullable enable
+
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage.MessageInterception
+{
+    /// <summary>
+    /// Specifies what LSP method the <see cref="MessageInterceptor"/> handles. May be applied multiple times.
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    internal class InterceptMethodAttribute : MultipleBaseMetadataAttribute
```

Comment:
I legit have never seen `MultipleBaseMetadataAttribute` used. Learn something new every day 😉

Reply:
I totally copied this from some of the other metadata attributes in VS. 🤣